src/client/runner/automation/utils.js

```diff
@@ -95,10 +91,11 @@ export function focusByRelatedElement (element) {
     if (!elementForFocus || domUtils.getActiveElement() === elementForFocus)
         return;
 
-    focusBlurSandbox.focus(elementForFocus, noop, false, true);
+    focusBlurSandbox.focus(elementForFocus, () => {
```

Comment:
define the empty function as `noop` in this file (like [here](https://github.com/DevExpress/testcafe/blob/28f103a19ef0e56ee2fcc0599ab5d320e1cdc15b/src/client/runner/automation/util.js#L4))